scenario/BulkerScenario.ts

```diff
@@ -332,3 +332,24 @@ scenario(
     return txn; // return txn to measure gas
   }
 );
+
+scenario(
+  'Comet#bulker > it reverts when passed an action that does not exist',
+  {},
+  async ({ comet, actors }) => {
+    const { betty } = actors;
+
+    const supplyGalacticCreditsCalldata = utils.defaultAbiCoder.encode(
+      ['address', 'address', 'uint'],
+      [comet.address, betty.address, exp(1, 18)]
+    );
+    const calldata = [supplyGalacticCreditsCalldata];
+    const actions = [
+      ethers.utils.formatBytes32String('ACTION_SUPPLY_GALACTIC_CREDITS')
```

Comment:
👽: Comet doesn't support Galactic Credits?!

Reply:
it's an outrage! just wait until the Emperor hears about this